v5/owner.apib

```diff
@@ -0,0 +1,449 @@
+# Group Owner
+
+## Owner [/owner]
+Access needs a __private key__.
+
+### List parking spaces [GET /owner/parking-spaces]
+Returns all parking spaces that are owned by the user given by the used private token for the request.
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+    + Attributes (Parking space)
+
+### Retrieve a Parking space [GET /owner/parking-spaces/{parking_space_id}]
+Retrieves the details of one singe parking space.
+
++ Parameters
+    + parking_space_id: `005c4826-4e28-11e3-a675-d43d7eece53d` (string, required) - The id or code of the parking space
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+    + Attributes (Parking space)
+
++ Response 404
+Returned if parking space was not found or it doesn't belong to the user.
+    + Body
+
+### Update a Parking space [PUT /owner/parking-spaces/{parking_space_id}]
+Updates the parking space by setting the values of the parameters passed. Any parameter not provided will be left unchanged.
+
++ Parameters
+    + parking_space_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request  (application/json)
+    + Attributes
+        + `status`: `inactive` (string)
```

Comment:
here you should mention all possible values, search for `+ Members` inside the project to find examples (e.g. List bookings for v5)

Reply:
I should have mentioned that earlier, but the exact format of the documentation is Api Blueprint and you can find the specification here: https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md

this includes these more advanced things like the Members syntax

Reply:
Done